x/concentrated-liquidity/pool_hooks.go

```diff
@@ -43,29 +32,7 @@ func (k Keeper) AfterCreatePosition(ctx sdk.Context, poolId uint64, owner sdk.Ac
 	if err != nil {
 		return err
 	}
-	return k.callPoolActionListener(ctx, msgBz, poolId, afterActionPrefix(types.CreatePositionPrefix))
-}
-
-// BeforeAddToPosition is a hook that is called before liquidity is added to a position.
-func (k Keeper) BeforeAddToPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, positionId uint64, amount0Added osmomath.Int, amount1Added osmomath.Int, amount0Min osmomath.Int, amount1Min osmomath.Int) error {
```

Comment:
`AddToPosition` hooks were removed since the function just calls `WithdrawPosition` then `CreatePosition` under the hood. This makes it tricky to separate out `AddToPosition` hooks from the other two and imo unnecessary complexity that would increase the surface for circumventing hooks.

Opted to remove and keep hooks on the most primitive actions.